.vitepress/config.js

```diff
@@ -0,0 +1,381 @@
+                for (let editorType in editordata) {
+                    for (let editor of editordata[editorType]) {
+                        if (editorUsernames.includes(editor)) continue;
+                        if (editorType === 'inactive') {
+                            inactiveEditorUsernames.push(editor);
+                        } else {
+                            editorUsernames.push(editor);
+                        }
+                    }
+                }
+                for (let username of editorUsernames) {
+                    let editorTypes = [];
+                    for (let editorType in editordata) {
+                        if (editordata[editorType].includes(username)) {
+                            editorTypes.push(editorType.charAt(0).toUpperCase() + editorType.slice(1));
+                        }
+                    }
+                    editors.push({
+                        avatar: `https://github.com/${username}.png`,
+                        name: username,
+                        title: editorTypes.join(', '),
+                        links: [
+                            { icon: 'github', link: `https://github.com/${username}` }
+                        ]
+                    });
+                }
+                for (let username of inactiveEditorUsernames) {
+                    emeritusEditors.push({
+                        avatar: `https://github.com/${username}.png`,
+                        name: username,
+                        title: 'Emeritus Editor',
+                        links: [
+                            { icon: 'github', link: `https://github.com/${username}` }
+                        ]
+                    });
+                }
+                frontmatter.editors = editors;
+                frontmatter.emeritusEditors = emeritusEditors;
+            }
+
+            if (frontmatter.eip) {
+                // Try to read from cache
+                try {
+                    let cache = JSON.parse(fs.readFileSync(`./.vitepress/cache/eips/${frontmatter.eip}.json`));
+                    frontmatter = { ...frontmatter, ...cache };
+                } catch (e) {
+                    logger.info(`Cache miss for ${pageData.relativePath}`);
+                }
+                // The below caused so much pain and suffering :|
+                if (!frontmatter.created) {
+                    let initial = new Date((await git.log(['--diff-filter=A', '--', pageData.relativePath])).latest.date); // Only one match, so this is fine to use latest
+                    if (initial) {
+                        frontmatter.created = formatDateString(initial);
+                    }
+                }
+                if (!frontmatter.finalized && frontmatter.status === 'Final') {
+                    let final = new Date((await git.raw(['blame', pageData.relativePath])).split('\n').filter(line => line.match(/status:\s+final/gi))?.pop()?.match(/(?<=\s)\d+-\d+-\d+/g)?.pop());
```

Comment:
You should extract the `git blame` parsing into its own function, since you use it a few times.

Reply:
Will be fixed in the next commit.